engine/runtime/src/test/scala/org/enso/compiler/test/context/SuggestionBuilderTest.scala

```diff
@@ -1506,7 +1493,7 @@ class SuggestionBuilderTest extends CompilerTest {
                   .Argument("this", "Unnamed.Test.Cons", false, false, None)
               ),
               selfType      = "Unnamed.Test.Cons",
-              returnType    = "List",
+              returnType    = retType,
```

Comment:
Instead of `List` string name we get fully qualified `Sum` type with list of constructors that may be used to construct it. Good, that's what we wanted.